### PR TITLE
Setup Exhale for ReadTheDocs Documentation generation

### PR DESCRIPTION
**Pull request by cpyarger**
This will allow for automated documentation building to be hosted at the following places.
https://libremidi.readthedocs.io/  
http://libremidi.rtfd.io/

The current version on RTD is pointed at my fork, but if accepted I will point it at this one. 
I have also added you as a maintainer of the documentation on RTD and can look into transferring ownership to you.

### CODE REVIEW

**Comment by cpyarger** [comment]
This is to help support #18

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import sys
+import textwrap
+
+#sys.path.insert(0, os.path.abspath('.'))
+#import exhale
+
+# -- Project information -----------------------------------------------------
+
+project = 'libremidi'
+copyright = '2021, Jean-Michaël Celerier'
```
Comment (jcelerier) [inline docs/conf.py:23]: s/my name/libremidi authors/

**Review by jcelerier — APPROVED** [review]
looks pretty good, thanks for doing that ! do you think there's an  easy way to add the .md files to the doc too ?

**Comment by jcelerier** [comment]
I had kinda forgotten about this - what are the readthedocs credentials if I want to update the docs?

**Pull request merged**